docs/scripts/check-closing-quotes.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Check for invalid quote usage such as "variable `x``"
+
+DIR="$1"
+WHITELIST="$2"
```

Comment:
What do you think about not calling this "whitelist" but rather something like "allowlist"?

Reply:
oh you're right, eliminating inappropriate naming.

Reply:
done

Reply:
Awesome. Thanks a lot!